cmd/kind/get/kubeconfigpath/kubeconfigpath.go

```diff
@@ -52,6 +52,15 @@ func NewCommand() *cobra.Command {
 }
 
 func runE(flags *flagpole) error {
+	// Check if the cluster name exists
+	known, err := cluster.IsKnown(flags.Name)
```

Comment:
should we chick is known?

Reply:
no, the path is fixed regardless of the cluster.